ftfy/cli.py

```diff
@@ -17,17 +37,62 @@ def main():
     """
     import argparse
 
-    parser = argparse.ArgumentParser()
-    parser.add_argument('filename', help='file to transcode')
+    parser = argparse.ArgumentParser(
+        description="ftfy (fixes text for you), version %s" % __version__
+    )
+    parser.add_argument('filename', default='-', nargs='?',
+                        help='The file whose Unicode is to be fixed. Defaults '
+                             'to -, meaning standard input.')
+    parser.add_argument('-g', '--guess', action='store_true',
+                        help="Ask ftfy to guess the encoding of your input. "
+                             "This is risky. Overrides -e.")
+    parser.add_argument('-e', '--encoding', type=str, default='utf-8',
+                        help='The encoding of the input. Defaults to UTF-8.')
+    parser.add_argument('-n', '--normalization', type=str, default='NFC',
+                        help='The normalization of Unicode to apply. '
+                             'Defaults to NFC. Can be "none".')
+    parser.add_argument('--preserve-entities', action='store_true',
+                        help="Leave HTML entities as they are. The default "
+                             "is to decode them in non-HTML files.")
```

Comment:
This implies that it's looking at the file extension or the file structure or something (which in this case it probably could do and might be a good idea), but really it's still just looking for angle-brackets, right?